app/fissile.go

```diff
@@ -951,6 +951,14 @@ func (f *Fissile) generateHelmValues(outputDir string, rolesManifest *model.Role
 		_, err = outputFile.Write(buf)
 	}
 	if err == nil {
+		_, err = outputFile.Write([]byte(`kube:
```

Comment:
I was thinking the first option, mainly due to indenting issues with the literal YAML.  (But without the `k8s` thing all on one line, probably…)

If you feel strongly about it, though, we can go with what you have.